normandy/recipes/tests/test_models.py

```diff
@@ -208,20 +208,47 @@ def test_matches_deals_with_none(self):
 
 
 class TestClient(object):
-    @pytest.mark.parametrize('x_forwarded_for,expected_ip', [
-        ('192.0.2.0,127.0.0.1', '192.0.2.0'),
-        ('192.0.2.0', '192.0.2.0'),
-    ])
-    def test_country_x_forwarded_for(self, rf, x_forwarded_for, expected_ip):
-        client = Client(rf.get('/', HTTP_X_FORWARDED_FOR=x_forwarded_for))
+    def test_country_remote_addr_fallback(self, rf):
```

Comment:
This test isn't necessary anymore, you're testing the same thing as when there are no proxies.